examples/02-plot/chart_overlays.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
Use sphinx references to allow for the automatic generation of links between the rendered documentation.

```suggestion
"""
.. _chart_overlays_example:
```